README.md

```diff
@@ -1,5 +1,10 @@
 # 🏗 Scaffold-ETH 2
 
+<h4 align="center">
+  <a href="https://docs.scaffoldeth.io">Documentation</a> |
```

Comment:
Until we update it should we add https://scaffold-eth-2-docs.vercel.app/ her for now ?